code/game/turfs/flooring/seasonal.dm

```diff
@@ -0,0 +1,216 @@
+var/season
```

Comment:
More obvious name for global var! current_season or world_time_season or something!